cmd/mt-fakemetrics/cmd/root.go

```diff
@@ -107,6 +109,8 @@ func init() {
 	rootCmd.PersistentFlags().StringSliceVar(&customTags, "custom-tags", []string{}, "A list of comma separated tags (i.e. \"tag1=value1,tag2=value2\")(default empty) conflicts with add-tags")
 	rootCmd.PersistentFlags().IntVar(&numUniqueCustomTags, "num-unique-custom-tags", 0, "a number between 0 and the length of custom-tags. when using custom-tags this will make the tags unique (default 0)")
 
+	rootCmd.PersistentFlags().StringVar(&valuePolicy, "value-policy", "", "a value policy (i.e. \"single:1\" \"multiple:1,2,3,4,5\" \"timestamp\")")
```

Comment:
you're adding this as a root flag, which implies it works for all sub-commands. but it seems to only work for feeds.

Reply:
So, the tags flags in root should also be moved to feed and backfill?